tests/test_idna_codec.py

```diff
@@ -53,7 +53,7 @@ def testIncrementalEncoder(self):
             self.assertEqual(b"".join(codecs.iterencode(decoded, "idna")),
```

Comment:
Is this part still necessary? `idna` does not appear to have a `iterencode` method.